### PR TITLE
Update pymaker for 3 new collaterals...

### PR DESCRIPTION
**Pull request by EdNoepel**
...and improve manual tests.

### CODE REVIEW

File: tests/manual_test_create_unsafe_vault.py [hunk 1]
```diff
@@ -64,6 +63,18 @@ def r(value, decimals=1):
 logging.info(f"{'':<7} duty={mcd.jug.duty(ilk)} min_amount={token.min_amount}")
 
 
+if osm_price == Wad(0) and isinstance(collateral.pip, OSM):
+    logging.warning("OSM price is 0; poking OSM")
+    osm: OSM = collateral.pip
+    osm.poke().transact()
```
Comment (EdNoepel) [inline tests/manual_test_create_unsafe_vault.py:69]: I know user might need to wait an hour and poke again to test, but better than nothing.

**Review by Exef — APPROVED** [review]
LGTM! 👍

File: tests/manual_test_create_unsafe_vault.py (hunk 1 above)
Comment (godsflaw) [reply tests/manual_test_create_unsafe_vault.py:69]: I love that you added this.

**Review by godsflaw — APPROVED** [review]
LGTM

**Review by KentonPrescott — APPROVED** [review]
LGTM

**Pull request merged**